Bluejay.asm

```diff
@@ -1789,9 +1789,10 @@ scheduler_steps_even:
 
 scheduler_steps_even_demag_metric_frame:
 	; Check telemetry is enable to produce telemetry frames
-	jb Flag_Ext_Tele, scheduler_steps_even_demag_metric_frame
+	jb Flag_Ext_Tele, scheduler_steps_even_demag_metric_frame_prepare
```

Comment:
Can we not do:
```
jnb Flag_Ext_Tele, scheduler_exit
```
instead?

Reply:
No, we can't: would be too long of a jump.

Thanks @damosvil via Discord.

Reply:
It is a long jump, jnb cannot jump that far